includes/Modules/Analytics.php

```diff
@@ -962,6 +962,27 @@ function ( $dimension_def ) {
 					}
 				}
 
+				$dimension_filters          = $data['dimensionFilters'];
+				$dimension_filter_instances = array();
+				if ( ! empty( $dimension_filters ) && is_array( $dimension_filters ) ) {
+					foreach ( $request_args['dimensions'] as $dimension ) {
+						$dimension_name = $dimension->getName();
+						if ( isset( $dimension_filters[ $dimension_name ] ) ) {
```

Comment:
Let's avoid the additional nesting here by using `continue` if the key isn't set.

Reply:
Updated.